sdk/resource/env.go

```diff
@@ -88,7 +89,12 @@ func constructOTResources(s string) (*Resource, error) {
 			invalid = append(invalid, p)
 			continue
 		}
-		k, v := strings.TrimSpace(field[0]), strings.TrimSpace(field[1])
+		k := strings.TrimSpace(field[0])
+		v, err := url.QueryUnescape(strings.TrimSpace(field[1]))
+		if err != nil {
```

Comment:
What do you think of invoking `otel.Handle(err)` if the `err` is non-nil?

Reply:
That's a good question and I wasn't sure how to handle it, so I opted to maintain the existing behaviour: if, for some reason, your env car has some invalid percent-encoded values things will continue to work after this change.

While this prioritizes backwards compatibility, it's a silent failure which may be hard to detect and fix.

Personally I'm good either way, so I am open to suggestions 🙂

Reply:
Depending what users do in `otel.Handle`, it's likely to remain a non-breaking change (people usually use Handle to report errors, not to kill the process). So I'd lean towards avoiding silent failures.

Reply:
Ah thank you, I didn't realize that's how the handler worked. PR updated :slightly_smiling_face: